libs/stocks/feature-shell/src/lib/stocks/stocks.component.ts

```diff
@@ -32,7 +32,9 @@ export class StocksComponent implements OnInit {
     });
   }
 
-  ngOnInit() {}
+  ngOnInit() {
+    this.stockPickerForm.valueChanges.subscribe(this.fetchQuote);
+  }
```

Comment:
it should be implemented as

`this.stockPickerForm.valueChanges.subscribe(() => {
        this.fetchQuote();
    });`

to prevent undefined error

Reply:
Also would recommend calling the function on out of focus instead of calling the function on every keypress.

Also need to show the backend error in case the symbol is invalid